### PR TITLE
FIX: targeting chat-message-text is more reliable

### PR DESCRIPTION
**Pull request by jjaffeux**
We were having flakey specs on mobile when clicking with delay on a message, it appears that sometimes the click was too much on a border of the container and was not triggering the event.

### CODE REVIEW

File: plugins/chat/spec/system/channel_members_page_spec.rb [hunk 1]
```diff
@@ -31,11 +31,12 @@
     end
 
     context "with members" do
+      fab!(:searched_user) { Fabricate(:user) }
+
       before do
         channel_1.add(current_user)
-        channel_1.add(Fabricate(:user, username: "cat"))
-        98.times { channel_1.add(Fabricate(:user)) }
-        Jobs.run_immediately!
```
Comment (jjaffeux) [inline plugins/chat/spec/system/channel_members_page_spec.rb:38]: Removing this is unrelated to the flakeyness, it would fail even when removed.

File: plugins/chat/spec/system/channel_members_page_spec.rb [hunk 2]
```diff
@@ -31,11 +31,12 @@
     end
 
     context "with members" do
+      fab!(:searched_user) { Fabricate(:user) }
+
       before do
         channel_1.add(current_user)
-        channel_1.add(Fabricate(:user, username: "cat"))
```
Comment (jjaffeux) [inline plugins/chat/spec/system/channel_members_page_spec.rb:36]: setting username or not is unrelated with flakeyness, it would fail when removed too

**Review by pmusaraj — APPROVED** [review]
